### ISSUE TITLE
recent tags appear to be pegged to the wrong SHA ID

### ISSUE THREAD

**Issue by TallTed** [tenforce/docker-virtuoso#24]
The "latest" image builds a 7.2.4.2 binary, not a 7.2.5.1

```pre
$ sudo docker run tenforce/virtuoso:latest virtuoso-t -?
Virtuoso Open Source Edition (Column Store) (multi threaded)
Version 7.2.4.2.3217-pthreads as of Nov  5 2018
Compiled for Linux (x86_64-pc-linux-gnu)
Copyright (C) 1998-2016 OpenLink Software
```

That should show `7.2.5.3229`.

Looks like the [dockerfile](https://hub.docker.com/r/tenforce/virtuoso/dockerfile) says --
```pre
# Set Virtuoso commit SHA to Virtuoso 7.2.4 release (25/04/2016)
ENV VIRTUOSO_COMMIT 96055f6a70a92c3098a7e786592f4d8ba8aae214
```
-- where it should say --
```pre
# Set Virtuoso commit SHA to Virtuoso 7.2.5.1 release (2018-08-15)
ENV VIRTUOSO_COMMIT 17c4ba1d5825822d8380ef8d9f978e2d57f4768f
```

**Comment by TallTed** [comment]
It looks like the [7.2.5 branch was created](https://github.com/tenforce/docker-virtuoso/tree/virtuoso7.2.5), but not merged to [master](https://github.com/tenforce/docker-virtuoso/tree/master), so the auto-built docker image remains at 7.2.4 ...

**Comment by TallTed** [comment]
@nvdk @sandervd @wdullaer @jvstein ???

**Comment by TallTed** [comment]
Fixed by 550b325f5e410cc660295c6a8ab5c05f74476fa3
